product-specifications/example-2.md

```diff
@@ -0,0 +1,21 @@
+# Interactive Animal Identification Tool (IAIT)
+
+This tool will help users identify animals by asking a series of questions and then provide a list of most likely animals based on the user's responses. This tool will be used by wildlife conservationists to identify animals they encounter.
+
+## User Flow
+
+1. Users answer a series of questions about the observed animal:
+    - What color is the animal?
+    - Where was the animal observed?
+    - How large was the animal?
+3. Based on user inputs, the system will analyze potential matches.
```

Comment:
should we probably have a text area for the user to describe any other features they feel would be important in identifying the animal.